src/protocol/primitives/record-batch/index.ts

```diff
@@ -1,3 +1,3 @@
 export * from './attributes';
-export * from './record';
+export { Record, RecordHeader } from './record';
```

Comment:
Is there any reason why the `RecordHeader` class is not in a separate file?

Reply:
All 4 classes are only used within the Record class and I don't want to have them in separate files because it gets bloated.

Reply:
When class is exported, I believe it should be in a separate file. Smaller files are more readable and the code structure can be easier maintained.